homework_3/barbhaiwala/solution.js

```diff
@@ -0,0 +1,83 @@
+function NaiveCalculator(equation)
```

Comment:
Typically, I would only upper-case constructor function names. So, this would be better as `function naiveCalculator`